dev/release/binary-task.rb

```diff
@@ -610,7 +610,7 @@ def download_url(url, output_path)
               $stderr.puts(build_download_error_message(url, response))
               return
             else
-              raise message
```

Comment:
@kou The original error was caused by undefined var `message`, but once I re-raised the exception the script was also exiting because certain kind of http error was not being handled so I worked around the problem by not re-raising just unconditionally retrying.

Reply:
Oh, sorry.
We should show request/response details to know what error is occurred.

I've pushed a commit that raises an exception with the request/response details.
We'll be able to know what response should be handled here to retry download. Then we can improve our retry code.